libvirt/README.md

```diff
@@ -1,6 +1,6 @@
 # Machine Stats for Virtual environments (Virt-Stats)
```

Comment:
I think we need a new title. Given that the rest of machine stats, the ansible and powershell parts, work with virtual environments as well. I don't think it's clear when someone would use this or use the other workflows.

Maybe `Machine Stats for Hypervisors` or `Machine Stats via Hypervisors`.

Reply:
I like “Machine Stats for Hypervisors”